test/it/uk/gov/hmrc/bindingtariffclassification/component/CaseSpec.scala

```diff
@@ -105,6 +106,7 @@ class CaseSpec extends BaseFeatureSpec {
       val responseCase = Json.parse(result.body).as[Case]
       responseCase.reference shouldBe "504400001"
       responseCase.status shouldBe CaseStatus.NEW
+      responseCase.application.asInstanceOf[BTIApplication].importOrExport shouldBe None
```

Comment:
Might be nice to have an `asBTI` method on `Application`

Reply:
I was looking for (and surprised that I couldn't find) a method like that.  I'll add one

Reply:
Done